inc/container.class.php

```diff
@@ -963,7 +963,7 @@ function getTabNameForItem(CommonGLPI $item, $withtemplate = 0) {
                   if (!$item->isEntityAssign() || in_array($item->fields['entities_id'], $entities)) {
 
                      $display_condition = new PluginFieldsContainerDisplayCondition();
-                     if($display_condition->computeDisplayContainer($item, $data['id'])) {
+                     if($display_condition->computeDisplayContainer($item, $data['id']) && !$withtemplate) {
```

Comment:
Maybe you should move this condition in the begining of the method. There is no need to compute anything if we already know that we will not add any tab on item.
```php
function getTabNameForItem(CommonGLPI $item, $withtemplate = 0) {
   if (!$withtemplate) {
      return [];
   }
   ...
}
```

Reply:
done